examples/image-streams/image-streams-centos7.json

```diff
@@ -635,13 +635,13 @@
             },
             "from": {
               "kind": "ImageStreamTag",
-              "name": "1"
+              "name": "2"
```

Comment:
Not only adding v2 jenkins, but defaulting to v2 Jenkins in the image stream .... yeah, I suppose it is time